WalletWasabi.Fluent/ViewModels/Wallets/Send/LabelSelectionViewModel.cs

```diff
@@ -53,6 +63,16 @@ public Pocket[] AutoSelectPockets(SmartLabel recipient)
 			return new[] { _privatePocket };
 		}
 
+		if (_semiPrivatePocket.Amount >= _targetAmount)
+		{
+			return new[] { _semiPrivatePocket };
+		}
+
+		if (privateAndSemiPrivatePockets.Sum(x => x.Amount) >= _targetAmount)
```

Comment:
Not introduced by this PR, but these equalities aren't considering transaction fees at all. This is bad.

Reply:
If I understand correctly the users can only select from these, so this will prevent them from making transactions altogether.

Reply:
Yes, that can happen and it will cause an exception in the TransactionPreviewViewModel which is handled there. Anyway, you are right, it can be improved.

Reply:
By "handled there" do you mean the user can make the transaction?

Reply:
Yes

Reply:
If that's correct, then nothing critical here.